### PR TITLE
add controller support

### PR DESCRIPTION
**Pull request by TheCrossBoy**
its basically already done how hard can it be

### CODE REVIEW

File: chariot-client/src/application.rs [hunk 1]
```diff
@@ -312,4 +315,126 @@ impl Application {
...
+                        Some(InputEvent::Engine(EngineStatus::Accelerating(value)))
+                    }
+                    Button::RightTrigger2 => {
+                        println!("New gamepad event: {:?}", event);
+                        Some(InputEvent::Engine(EngineStatus::Neutral))
+                    }
+                    // Brake
+                    Button::LeftTrigger2 => None,
+
+                    /***** MENU *****/ // TODO: this is temporary. we need a real way to handle menu input w controllers
```
Comment (trulyronak) [inline chariot-client/src/application.rs:340]: dear god i completely forgot about this LOL

File: chariot-client/src/application.rs [hunk 2]
```diff
@@ -312,4 +315,126 @@ impl Application {
...
+                    }
+                    _ => None,
+                }
+            }
+            // Axis (value: [-1, 1])
+            Err((axis, value)) => match axis {
+                /***** MOVEMENT *****/
+                // Turn right
+                Axis::LeftStickX if value > 0.0 => {
+                    println!("New gamepad event: {:?}", event);
```
Comment (trulyronak) [inline chariot-client/src/application.rs]: can you omit the printlns for gamepad events if we do something with it already

**Review by trulyronak — APPROVED** [review]
lgtm! just left a comment on removing unneeded printlns

File: chariot-client/src/application.rs (hunk 1 above)
Comment (TheCrossBoy) [reply chariot-client/src/application.rs:340]: I think for now we can just like... ignore it lol

We can add it later if we want to lmfao

**Pull request merged**